class RobotTerminatedError(Exception):
    exit(1)
